### PR TITLE
Bump version to 1.3.35 and enable Windows build.

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

* Enabling Windows port.
* Adding myself as co-maintainer.

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
Hmm, it is surprised to see that Linux build failed. Hope someone on gitter can shed some lights. https://gitter.im/conda-forge/conda-forge.github.io?at=5e83f340c6e44255a1c930c1

**Comment by seanyen** [comment]
@conda-forge/graphicsmagick The Windows port is enabled. However I don't know why the Linux build is broken, honestly I didn't change the Linux recipe. Hope someone can shed some lights here.

**Comment by wolfv** [comment]
Hmm I am wondering if somehow some unicode character slipped into any of the files? It seems to fail when the info data is bundled (that data also consists of the meta.yaml file and the build script).
The error says that characters from a certain unicode range are not allowed ... I checked briefly but couldn't spot anything out of the ordinary.
Maybe there is some tool to check if there are e.g. invisible unicode characters?

**Comment by seanyen** [comment]
Got a Linux box and run the recipe. It turned out `GraphicsMagick` created some weird file with funny characters under $PREFIX\lib and make it a symlink to `libxkbcommon.so`. I don't know where `GraphicsMagick` generates the symlink but if there is no `x11` related package, this problem will be gone so I try to restrict the `x11` to `osx` only.

image

**Comment by seanyen** [comment]
@conda-forge/graphicsmagick This is ready for review and merge. Thanks.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@bgruening Hi! Friendly ping. Hope we can add Windows build too.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cmake  # [win]
+    - ninja  # [win]
   host:
-    - ghostscript
+    - ghostscript  # [unix]
+    - gnuplot  # [osx]
```
Comment (wolfv) [inline recipe/meta.yaml:25]: is this correct? only OS X for gnuplot?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
+    - libwebp  # [unix]
+    - xorg-libsm  # [osx]
```
Comment (wolfv) [inline recipe/meta.yaml:27]: not sure this is correct for xorg on Linux ... 

maybe adding `libxkbcommon` would also fix the weird unicode issue?
https://anaconda.org/conda-forge/libxkbcommon

File: recipe/meta.yaml (hunk 1 above)
Comment (seanyen) [reply recipe/meta.yaml:25]: yeah, unfortunately, the `x11` packages are deps from `gnuplot` so I need to disable it for `linux`. :(

File: recipe/meta.yaml (hunk 2 above)
Comment (seanyen) [reply recipe/meta.yaml:27]: `libxkbcommon` is already in the dependency graph originally. Another idea is to disable the `Linux` build for `1.3.35` entirely. There is already a `Linux` build for `1.3.33` and the community still can make use of it. And we can tackle `Linux` build later. cc @bgruening

Btw, this is a dep for [`navigation2`](https://ros-planning.github.io/navigation2/) on `ros2`. JFYI.

**Comment by seanyen** [comment]
cc @isuruf This is ready for review and merge. Hope we can add a Windows port for it.

**Comment by seanyen** [comment]
@conda-forge/staged-recipes @bgruening Friendly ping! Hope we can move forward with a Windows build.

**Comment by isuruf** [comment]
For the linux issue. You can delete that weird file at the end of `build.sh`.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge/core @conda-forge/graphicsmagick This is ready for review and merge. Thanks!

**Comment by isuruf** [comment]
Can you fix the warnings in conda-build?

**Comment by seanyen** [comment]
@isuruf  Assuming you are talking about the `overlinking` and `overdepending`, they are fixed now.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,59 +1,68 @@
...
+    - ghostscript  # [unix]
+    - gnuplot  # [unix]
+    - libwebp  # [unix]
+    - libxml2  # [unix]
+    - zstd  # [unix]
+    - xorg-libsm  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libx11  # [linux]
+    - xorg-libice  # [linux]
+    - _openmp_mutex  # [linux]
```
Comment (isuruf) [inline recipe/meta.yaml:57]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,59 +1,69 @@
...
+    - ghostscript  # [unix]
+    - gnuplot  # [unix]
+    - libwebp  # [unix]
+    - libxml2  # [unix]
+    - zstd  # [unix]
+    - xorg-libsm  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libx11  # [linux]
+    - xorg-libice  # [linux]
+    - libgomp      # [linux]
```
Comment (isuruf) [inline recipe/meta.yaml:58]: Please remove this line. Only build is needed.

**Comment by seanyen** [comment]
@isuruf Thanks for the feedback. Let me know if any additional things to fix.

**Comment by seanyen** [comment]
@conda-forge/core @conda-forge/graphicsmagick Sorry to spam again. This is ready for review and merge. Thanks!

**Pull request merged**